recipes/blast-legacy/2.2.26/build.sh

```diff
@@ -0,0 +1,7 @@
+cp -R bin $PREFIX
+cp -R data $PREFIX
+cp -R doc $PREFIX
```

Comment:
is doc and data needed?

Reply:
These were taken from the `build.sh` from the existing `blast-legacy` 2.2.22 recipe. The doc for standalone `blast` refers to the `data` directory but it needs the user to configure the location by creating a  `.ncbirc` file. I'm happy to remove `data` and `doc`if they're not appropriate in your opinion.

Reply:
I'm just wondering and would like to keep the package small if possible. If there is nothing that is relevant for the binaries, lets remove it.

Reply:
No problem, I understand. I'll remove them and if it turns out there's a problem then it's always possible to update in future. Thanks for taking a look!